test/Microsoft.ML.Predictor.Tests/TestEstimators.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Api;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.EntryPoints;
+using Microsoft.ML.Runtime.Learners;
+using Microsoft.ML.Runtime.RunTests;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.ML.Tests
+{
+    public class SimpleEstimatorTests : TestDataPipeBase
```

Comment:
>SimpleEstimatorTests [](start = 17, length = 20)

The filename is overly broad. Let's make it something like `TestTrainerEstimators`